Cargo.toml

```diff
@@ -41,6 +41,7 @@ version = "3.0.0"
 hex = "0.4.2"
 picky = "5.0.0"
 psa-crypto = { version = "0.2.0" , default-features = false, features = ["with-mbed-crypto"], optional = true }
+secrecy = { version = "0.6.0", features = ["serde"] }
```

Comment:
The interface already exports a version of this crate (`parsec_interface::secrecy`) - it's better to use that one to avoid potential version mismatches in the future.